fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -121,7 +121,6 @@ def run
           else
             is_swift_fastfile = args.include?("swift")
             Fastlane::Setup.new.run(user: options.user, is_swift_fastfile: is_swift_fastfile)
-            Fastlane::SwiftLaneManager.first_time_setup
```

Comment:
I moved these calls into `Fastlane::Setup` to (1) reduce duplication, and (2) avoid a crash if `Fastlane::Setup` returns when a project is already configured, but then `first_time_setup` runs anyway